security.md

```diff
@@ -0,0 +1,127 @@
+# Security
+
+## Reporting Security Issues
+
+When reporting a security issue, do not create an issue or file a pull
+request on GitHub. Instead, disclose the issue responsibly by sending an email
+to security@opencontainers.org (which is inhabited only by the maintainers of
+the various OCI projects).
+
+The maintainers take security seriously. If you discover a security issue,
+please bring it to their attention right away!
+
+## Disclosure Distrobution List
+
+This list is used to provide actionable information to multiple distribution vendors at once.
+This list is not intended for individuals to find out about security issues.
+
+### Embargo Policy
+
+The information members receive on security-announce@opencontainers.org must not be made public, shared, nor even hinted at anywhere beyond the need-to-know within your specific team except with the list's explicit approval.
+This holds true until the public disclosure date/time that was agreed upon by the list.
+Members of the list and others may not use the information for anything other than getting the issue fixed for your respective distribution's users.
+
+Before any information from the list is shared with respective members of your team required to fix said issue, they must agree to the same terms and only find out information on a need-to-know basis.
+
+In the unfortunate event you share the information beyond what is allowed by this policy, you must urgently inform the security@opencontainers.org mailing list of exactly what information leaked and to whom.
+A retrospective will take place after the leak so we can assess how to not make the same mistake in the future.
+
+If you continue to leak information and break the policy outlined here, you will be removed from the list.
+
+### Contributing Back
+
+This is a team effort. As a member of the list you must carry some water. This
+could be in the form of the following:
+
+**Technical**
+
+- Review and/or test the proposed patches and point out potential issues with
+  them (such as incomplete fixes for the originally reported issues, additional
+  issues you might notice, and newly introduced bugs), and inform the list of the
+  work done even if no issues were encountered.
+
+**Administrative**
+
+- Help draft emails to the public disclosure mailing list.
+- Help with release notes.
+
+### Membership Criteria
+
+To be eligible for the security-announce@opencontainers.org mailing list, your
+distribution should:
+
+0. Have an actively monitored security email alias for our project.
+1. Have a user base not limited to your own organization.
+2. Have a publicly verifiable track record up to present day of fixing security
+   issues.
+3. Not be a downstream or rebuild of another distribution.
+4. Be a participant and active contributor in the community.
```

Comment:
I'm not sure this is a reasonable restriction -- since I'm pretty sure there are distributions like Debian or Arch Linux that don't match this requirement.

Reply:
that's fair